src/problem_bank_scripts/problem_bank_scripts.py

```diff
@@ -1018,17 +1054,56 @@ def str_presenter(dumper, data2):
             encoding="utf8",
         )
 
-    # Move image assets
+    # Create the file errors list
+    os_errors = []
+
+    # Move client assets (generally images)
     files_to_copy = header.get("assets")
     if files_to_copy:
-        [copy2(pathlib.Path(source_filepath).parent / fl, output_path.parent) for fl in files_to_copy]
+        pl_path = output_path.parent
+        for file in files_to_copy:
+            try:
+                copy2(pathlib.Path(source_filepath).parent / file, pl_path / file)
+            except (FileExistsError, FileNotFoundError, IsADirectoryError, PermissionError) as e:
+                os_errors.append(str(e))
+
+    # Move server assets
+    files_to_copy = header.get("serverFiles")
+    if files_to_copy and instructor:
+        pl_path = output_path.parent
+        for file in files_to_copy:
+            try:
+                copy2(pathlib.Path(source_filepath).parent / file, pl_path / file)
+            except (FileExistsError, FileNotFoundError, IsADirectoryError, PermissionError) as e:
+                os_errors.append(str(e))
```

Comment:
Still working on coming up with a suitable test case for this (I have an idea or two), but since this would be the only of the 4 special directories for a question, it felt right to add it in here

I also figured that since these files wouldn't be visible to students, they probably shouldn't show up on the public version of generated files

Reply:
makes sense, I think this is an ok place for it to go.